demos/pkey/Makefile

```diff
@@ -0,0 +1,20 @@
+#
+# To run the demos when linked with a shared library (default):
+#
+#    LD_LIBRARY_PATH=../.. ./EVP_PKEY_EC_keygen
```

Comment:
Would `../../util/wrap.pl EVP_PKEY_EC_keygen` make more sense?  This is doing more than just setting the LD_LIBRARY_PATH.

Reply:
Maybe.. Probably better to do that as a seperate pass over all the demos